07_Bootstrap/praktikum/index.html

```diff
@@ -0,0 +1,137 @@
+                <a class="nav-link active" aria-current="page" href="#">Home</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">About</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Features</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Team</a>
+              </li>
+              <li class="nav-item dropdown">
+                <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+                  Dropdown link
+                </a>
+                <ul class="dropdown-menu">
+                  <li><a class="dropdown-item" href="#">Action</a></li>
+                  <li><a class="dropdown-item" href="#">Another action</a></li>
+                  <li><a class="dropdown-item" href="#">Something else here</a></li>
+                </ul>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Pricing</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+      <section class="main">
+        <div class="container">
+            <h1 class="main-title mt-5">Simple Systeme ForDiscount</h1>
+            <p class="main-subitle text-center text-secondary fw-light">Integer Cursus Bibendum Augue Ac Cursus.</p>
+            <div class="row content">
+                <div class="col-lg-4 text-center mb-sm-5">
+                    <img src="assets/cloud.svg" alt="" width="50">
+                    <h6 class="content-title my-3">INTRODUCING WHATSAPP</h6>
+                    <p class="content-subtitle mb-4 text-secondary fw-light">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, distinctio!</p>
+                    <a href="#" class="content-link text-secondary text-decoration-none">Read More</a>
+                </div>
+                <div class="col-lg-4 text-center mb-sm-5">
+                    <img src="assets/planet.svg" alt="" width="50">
+                    <h6 class="content-title my-3">USER FRIENDLY INTERFACE</h6>
+                    <p class="content-subtitle mb-4 text-secondary fw-light">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, distinctio!</p>
+                    <a href="#" class="content-link text-secondary text-decoration-none">Read More</a>
+                </div>
+                <div class="col-lg-4 text-center mb-sm-5">
+                    <img src="assets/asteroid.svg" alt="" width="50">
+                    <h6 class="content-title my-3">BUILD THE APP EVERYONE LOVE</h6>
+                    <p class="content-subtitle mb-4 text-secondary fw-light">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eligendi, distinctio!</p>
+                    <a href="#" class="content-link text-secondary text-decoration-none">Read More</a>
+                </div>
```

Comment:
kalo pengen pake symantic html untuk box image bisa pake figure > img > figcaption

reference:
[HTML `<figcaption>` Tag - w3school](https://www.w3schools.com/tags/tag_figure.asp)
[`<figcaption>`: The Figure Caption element - mdn web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figcaption)